lib/web_ui/lib/src/engine/clipboard.dart

```diff
@@ -26,10 +27,15 @@ class ClipboardMessageHandler {
       } else {
         callback(codec.encodeErrorEnvelope(
             code: 'copy_fail', message: 'Clipboard.setData failed'));
+        errorEnvelopeEncoded = true;
```

Comment:
There's no error thrown here. Why would the `catchError` callback be invoked after this?

Reply:
When you do the error reply, the code that handles it could crash.